torch/_refs/__init__.py

```diff
@@ -5114,6 +5114,22 @@ def bucketize(
     return start.to(dtype=out_dtype)
 
 
+@register_decomposition(aten.geometric)
+@out_wrapper()
+@elementwise_type_promotion_wrapper(
+    type_promoting_args=("self",),
+    type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.DEFAULT,
+)
+def geometric(self, p, generator=None):
+    assert generator is None
+    return self.copy_(
```

Comment:
this is wrong, `aten.geometric` is a non-inplace op, the inplace variant will be added with `_make_inplace`, so there should be no `copy_` here. 
@lezcano should there be a test to flag this kind of wrong behavior?

Reply:
It'd be great to have such a test. Let me see if I can put something of that sort next week. This should be a good complement for https://github.com/pytorch/pytorch/pull/92003

Reply:
Addressed in https://github.com/pytorch/pytorch/pull/92243

Reply:
Oops, thanks for the catch, done.